content/commands/ft.info/index.md

```diff
@@ -29,140 +29,326 @@ syntax_str: ''
-
-[Examples](#examples)
+Returns information and statistics about a given index.
 
 ## Required arguments
 
-<details open>
-<summary><code>index</code></summary>
+`index`
+<br />
+is the name of the given index. You must first create the index using [`FT.CREATE`]({{< baseurl >}}/commands/ft.create/).
 
-is full-text index name. You must first create the index using [`FT.CREATE`]({{< baseurl >}}/commands/ft.create/).
-</details>
+## RESP reply
+
+`FT.INFO` returns an array reply with pairs of keys and values.
+
+## Returned values
+
+### General
+
+| Return field name | Definition |
+|:---               |:---        |
+| `index_name` | The index name that was defined when index was created. |
+| `index_options` | The index options selected during `FT.CREATE` such as `FILTER {filter}`, `LANGUAGE {default_lang}`, etc. |
+| `index_definition` | Includes `key_type`, hash or JSON; `prefixes`, if any; and `default_score`. |
+| `attributes` | The index schema field names, types, and attributes. |
+| `num_docs` | The number of documents. |
+| `max_doc_id` | The maximum document ID. |
+| `num_terms` | The number of distinct terms. |
+| `num_records` | The total number of records. |
+
+### Various size statistics
+
+| Statistic | Definition |
+|:---       |:---        |
+| `inverted_sz_mb` | The memory used by the inverted index, which is the core data structure used for searching in RediSearch. The size is given in megabytes. |
+| `vector_index_sz_mb` | The memory used by the vector index, which stores any vectors associated with each document. |
+| `total_inverted_index_blocks` | The total number of blocks in the inverted index. |
+| `offset_vectors_sz_mb` | The memory used by the offset vectors, which store positional information for terms in documents. |
+| `doc_table_size_mb` | The memory used by the document table, which contains metadata about each document in the index. |
+| `sortable_values_size_mb` | The memory used by sortable values, which are values associated with documents and used for sorting purposes. |
+| `key_table_size_mb` | The memory used by the key table, which stores the mapping between document IDs and Redis keys. |
+| `geoshapes_sz_mb` | The memory used by GEO-related fields. |
+| `records_per_doc_avg` | The average number of records (including deletions) per document. |
+| `bytes_per_record_avg` | The average size of each record in bytes. |
+| `offsets_per_term_avg` | The average number of offsets (position information) per term. |
+| `offset_bits_per_record_avg` | The average number of bits used for offsets per record. |
 
-## Return
+### Indexing-related statistics
 
-FT.INFO returns an array reply with pairs of keys and values.
+| Statistic | Definition |
+|:---       |:---        |
+| `hash_indexing_failures` | The number of failures encountered during indexing. |
+| `total_indexing_time` | The total time is taken for indexing in seconds. |
+| `indexing` | Indicates whether the index is currently being indexed. |
```

Comment:
Looks a bit odd like that. Maybe "Indicates whether the data is currently being indexed" is another option.
```suggestion
| `indexing` | Indicates whether the index is currently being generated. |
```